src/App.css

```diff
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'PT Sans', sans-serif;
+}
+
+.home,
+.calc,
+.qoute {
+  display: flex;
+  height: 70vh;
+  align-items: center;
+  justify-content: center;
+  font-size: 1.5rem;
+}
+
+.calc {
+  display: flex;
+  flex-direction: row;
+  color: #000;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 16px;
+  margin: 2%;
+}
+
+.calcontent {
+  width: 50%;
+  padding: 10px;
+  margin: 1px;
+}
+
+.content {
+  border: 2px solid rgb(108, 108, 108);
+  background-color: #e0e0e0;
+  color: #858694;
+  margin: 1%;
+}
+

```

Comment:
- [ ] Let us improve as well as modify our styling to have great project, I suggest you to change the height of calculator as well as display as flex with the texts `Lets do some calculations` that are displayed on the left.
  
![react15](https://user-images.githubusercontent.com/55635977/130820885-ac1960c8-b042-4a84-89fe-f4ab92239fdf.png)